Build/NuGet/.pack-version

```diff
@@ -0,0 +1 @@
+2.0.2-pre
```

Comment:
@dilijev I simply chose that value for internal testing... Is there a version you prefer?